DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -322,7 +322,9 @@ private function loadMidgard2Session(array $session, ContainerBuilder $container
     private function odmLoad(array $config, ContainerBuilder $container)
     {
         $this->loader->load('odm.xml');
+        $dm = $container->getDefinition('doctrine_phpcr.odm.document_manager.abstract');
```

Comment:
you could move this into the `if` below, where you check whether you need to add the method call.

or better, move all locale handling into its own method, and `return` early if neither explicit service nor locales are configured.